docs/guide/start/managing-cli.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Monitoring and Managing Applications
+title_in_menu: Monitoring and Managing Applications
+layout: website-normal
+menu_parent: index-cli.md
+children:
+- { section: Applications } 
+- { section: Entities } 
+- { section: Sensors  } 
+- { section: Effectors  } 
+- { section: Activities } 
+---
+
+
+
+So far we have touched on Brooklyn's ability to *deploy* an application blueprint to a cloud provider, but this just 
+the beginning. The sections below outline how to manage the application that has been deployed.
+
+## Applications
+
+Having created the application we can query its status.  We can find a summary of all deployed apps:
+{% highlight bash %}
+$ br app
+ Id         Name     Status    Location   
+ hTPAF19s   Tomcat   RUNNING   ajVVAhER  
+{% endhighlight %}
+
+or the details of a given app.  The ID, hTPAF19s, can also be used instead of the name "Tomcat".
+{% highlight bash %}
+$ br app Tomcat
+  Id:              hTPAF19s   
+  Name:            Tomcat   
+  Status:          RUNNING   
+  ServiceUp:       true   
+  Type:            org.apache.brooklyn.entity.stock.BasicApplication   
+  CatalogItemId:   null   
+  LocationId:      ajVVAhER   
+  LocationName:    FixedListMachineProvisioningLocation:ajVV   
+  LocationSpec:    vagrantbyon   
+  LocationType:    org.apache.brooklyn.location.byon.FixedListMachineProvisioningLocation  
+{% endhighlight %}
+
+To ease management of multiple applications, or even to reduce the amount of typing required, it is convenient
```

Comment:
l43-49 can probably be dropped. It's a small digression from the flow that we don't revisit anywhere following.